### ISSUE TITLE
Question about concatenate patchs

### ISSUE THREAD

**Issue by hjwang-824** [QizeYang/PAUL#7]
The supervised local-based reid model usually divided the original image into m parts, and these local parts can be concatenated to reconstruct the original image. However, the M patchs sampled may not the original region of the image after affine transformation, concatenate them directly may not be able to reconstruct the original image, is it OK?

**Comment by QizeYang** [comment]
Yes, if you want to reconstruct the original image, it would be better to add a global branch. But for identification, some information is redundant.
